include/fcl/broadphase/detail/hierarchy_tree-inl.h

```diff
@@ -1095,7 +1095,7 @@ struct SelectImpl<S, AABB<S>>
     if(vel[1] > 0) bv.max_[1] += vel[1];
     else bv.min_[1] += vel[1];
     if(vel[2] > 0) bv.max_[2] += vel[2];
-    else bv.max_[2] += vel[2];
+      else bv.min_[2] += vel[2];
```

Comment:
I believe @mamoll actually meant to change `bv.max_[2]` to `bv.min_[2]` withouth changing the indentation as:
```cpp
if(vel[2] > 0) bv.max_[2] += vel[2];
else bv.min_[2] += vel[2];
```